examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/Program.cs

```diff
@@ -39,11 +39,14 @@ public static void Main(string[] args)
                             builder.AddAzureAppConfiguration(options =>
                             {
                                 options.Connect(new Uri(appConfigurationEndpoint), new DefaultAzureCredential())
-                                       .Select(keyFilter: "WebDemo:*")
```

Comment:
Why delete this?

Reply:
Reset change.

---
In reply to: [433595829](https://github.com/Azure/AppConfiguration/pull/329#discussion_r433595829) [](ancestors = 433595829)